buf/registry/module/v1beta1/tag_service.proto

```diff
@@ -101,18 +101,26 @@ message ListTagsResponse {
 message CreateTagsRequest {
   // An individual request to create a Tag.
   message Value {
-    // The Module to create the Tag under.
-    ModuleRef module_ref = 1 [(buf.validate.field).required = true];
     // The Tag name.
-    string name = 2 [
+    //
+    // If there is already a Tag with a name, the request errors. Otherwise, a new Tag will be
```

Comment:
Ah this makes this no longer idempotent. @bufdev thoughts on this? It's also hard to make this idempotent now with the `ResourceRef` in play, because the resource may have changed.